src/incomingcallnotifications.cs

```diff
@@ -0,0 +1,114 @@
+//
+// IncomingCallNotifications C# bindings
+//
+// Authors:
+//	TJ Lambert  <t-anlamb@microsoft.com>
+//
+// Copyright 2019 Microsoft Corporation.
+//
+
+using System;
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+
+namespace IncomingCallNotifications {
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[Native]
+	enum ICHandleType : long {
+		Generic = 1,
+		PhoneNumber = 2,
+		EmailAddress = 3,
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[Native]
+	enum ICDeclineReason : long {
+		Generic,
+		Cancelled,
+		UserBusy,
+		Unanswered,
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface ICHandle : NSCopying {
+
+		[Export ("type")]
+		ICHandleType Type { get; }
+
+		[Export ("value", ArgumentSemantic.Copy)]
+		string Value { get; }
+
+		[DesignatedInitializer]
+		[Export ("initWithType:value:")]
+		IntPtr Constructor (ICHandleType type, string value);
+
+		[Export ("isEqualToHandle:")]
+		bool IsEqual (ICHandle handle);
+	}
+
+	[iOS (13,0), Mac (10, 15, onlyOn64: true)]
```

Comment:
Headers only seem to show this for:

```
+IC_CLASS_AVAILABLE(ios(13.0))
+@interface ICNotification : NSObject <NSCopying>
```
Did I miss something?

Reply:
@chamons The framework does exist in the SDK `/Applications/Xcode11-beta1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk/System/Library/Frameworks/IncomingCallNotifications.framework` and I can confirm that it exists in Catalina. It kind of makes sense since you can pair your Mac with your phone and answer calls in your computer or use FaceTime.

Reply:
I agree, but it's not in the headers fwict?

I hate to bind it and have them remove it later?

File radar and make iOS only? Or am I missing something?

Reply:
mmm since it makes sense I would leave as is but lets have some middle ground here we can file an issue about this and keep track of it in new betas, would this suggestion work for you?

Reply:
That's fine. Add it for now, add removing it in https://github.com/xamarin/xamarin-macios/issues/6212 if not in header. Maybe file a radar for bonus points?